### PR TITLE
feat: Add `battery` icons

### LINKED ISSUE #22: Icon request: intermediate battery levels
## Icon Request

* Icon name: battery-25, battery-50, battery-75, battery-full and -charging versions.
* Use case: There are currently 2 battery icons: one when plugged in ("battery-charging") and the regular "battery". It would be nice to be able to indicate the charge level as well. 0 (available currently), 25%, 50%, 75% and full should be enough.
* Screenshots of similar icons:

### PR DESCRIPTION
**Pull request by jletey**
localhost_3000__query=battery

Closes #22

### CODE REVIEW

**Comment by ericfennis** [comment]
Nice work, looks great! :fire::fire::fire:

**Comment by ericfennis** [comment]
maybe refactor the code with some basic svg elements instead of paths.

**Comment by jletey** [comment]
@ericfennis I'm not that familiar with SVG elements 😅 So I won't be able to do this very easily

**Comment by ericfennis** [comment]
I can help you, no problem. I will make some suggestions.

**Comment by jletey** [comment]
@ericfennis Would love that 👍

File: icons/battery-charging.svg [hunk 1]
```diff
@@ -9,7 +9,7 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M5 18H3a2 2 0 0 1-2-2V8a2 2 0 0 1 2-2h3.19M15 6h2a2 2 0 0 1 2 2v8a2 2 0 0 1-2 2h-3.19" />
-  <line x1="23" y1="13" x2="23" y2="11" />
-  <polyline points="11 6 7 12 13 12 9 18" />
+  <path d="M7 7H3.77778C2.79594 7 2 7.74619 2 8.66667V15.3333C2 16.2538 2.79594 17 3.77778 17H6M14 7H16.2222C17.2041 7 18 7.74619 18 8.66667V15.3333C18 16.2538 17.2041 17 16.2222 17H13" />
+  <path d="M10.6667 7L7.5 12H12.5L9.33333 17" />
+  <path d="M22 13V11" />
```
Comment (ericfennis) [inline icons/battery-charging.svg:14]:
```suggestion
  <path d="M7 7H3.77778C2.79594 7 2 7.74619 2 8.66667V15.3333C2 16.2538 2.79594 17 3.77778 17H6M14 7H16.2222C17.2041 7 18 7.74619 18 8.66667V15.3333C18 16.2538 17.2041 17 16.2222 17H13" />
  <polyline points="11 7 8 12 12 12 9 17" />
  <line x1="22" x2="22" y1="11" y2="13" />
```

File: icons/battery-full.svg [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M16.2222 7H3.77778C2.79594 7 2 7.74619 2 8.66667V15.3333C2 16.2538 2.79594 17 3.77778 17H16.2222C17.2041 17 18 16.2538 18 15.3333V8.66667C18 7.74619 17.2041 7 16.2222 7Z" />
+  <path d="M22 13V11" />
+  <path d="M6 10V14" />
+  <path d="M10 10V14" />
+  <path d="M14 10V14" />
```
Comment (ericfennis) [inline icons/battery-full.svg:16]:
```suggestion
  <rect x="2" y="7" width="16" height="10" rx="2" ry="2" />
  <line x1="22" x2="22" y1="11" y2="13" />
  <line x1="6" x2="6" y1="10" y2="14" />
  <line x1="10" x2="10" y1="10" y2="14" />
  <line x1="14" x2="14" y1="10" y2="14" />
```

File: icons/battery-low.svg [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M16.2222 7H3.77778C2.79594 7 2 7.74619 2 8.66667V15.3333C2 16.2538 2.79594 17 3.77778 17H16.2222C17.2041 17 18 16.2538 18 15.3333V8.66667C18 7.74619 17.2041 7 16.2222 7Z" />
+  <path d="M22 13V11" />
+  <path d="M6 10V14" />
```
Comment (ericfennis) [inline icons/battery-low.svg:14]:
```suggestion
  <rect x="2" y="7" width="16" height="10" rx="2" ry="2" />
  <line x1="22" x2="22" y1="11" y2="13" />
  <line x1="6" x2="6" y1="10" y2="14" />
```

File: icons/battery-medium.svg [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M16.2222 7H3.77778C2.79594 7 2 7.74619 2 8.66667V15.3333C2 16.2538 2.79594 17 3.77778 17H16.2222C17.2041 17 18 16.2538 18 15.3333V8.66667C18 7.74619 17.2041 7 16.2222 7Z" />
+  <path d="M22 13V11" />
+  <path d="M6 10V14" />
+  <path d="M10 10V14" />
```
Comment (ericfennis) [inline icons/battery-medium.svg:15]:
```suggestion
  <rect x="2" y="7" width="16" height="10" rx="2" ry="2" />
  <line x1="22" x2="22" y1="11" y2="13" />
  <line x1="6" x2="6" y1="10" y2="14" />
  <line x1="10" x2="10" y1="10" y2="14" />
```

File: icons/battery.svg [hunk 5]
```diff
@@ -9,6 +9,6 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <rect x="1" y="6" width="18" height="12" rx="2" ry="2" />
-  <line x1="23" y1="13" x2="23" y2="11" />
+  <path d="M16.2222 7H3.77778C2.79594 7 2 7.74619 2 8.66667V15.3333C2 16.2538 2.79594 17 3.77778 17H16.2222C17.2041 17 18 16.2538 18 15.3333V8.66667C18 7.74619 17.2041 7 16.2222 7Z" />
+  <path d="M22 13V11" />
```
Comment (ericfennis) [inline icons/battery.svg:13]:
```suggestion
  <rect x="2" y="7" width="16" height="10" rx="2" ry="2" />
  <line x1="22" x2="22" y1="11" y2="13" />
```

**Comment by jletey** [comment]
@ericfennis That's amazing! Thank you so much ❤️ Suggestions applied.

**Pull request merged**